src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -1384,7 +1384,7 @@ static class B
             // https://github.com/dotnet/roslyn/issues/52870: GetSymbolInfo() should return resolved method from method group.
             Assert.Null(symbolInfo.Symbol);
 
-            Assert.Equal(["void System.Object.F(System.Object y)", "void System.Object.F<T>(T y)"],
+            Assert.Equal((string[])["void System.Object.F(System.Object y)", "void System.Object.F<T>(T y)"],
```

Comment:
Is it worth just adding a polyfill `Assert.Equal<T>(ReadOnlySpan<T>, T[])` until we upgrade to 2.9.1 when it comes out?

Reply:
Wouldn't such a method need to be under AssertEx or similar?

FWIW, it might be good to solve it that way because we could delete that method when upgrading to 2.9.1, and then pretty easily revert all the usages to call the method from xunit.

Reply:
This would be my preference.